automatic/protoc/tools/chocolateyBeforeModify.ps1

```diff
@@ -0,0 +1,3 @@
+﻿Write-Host "Removing protoc shim before upgrading/uninstalling..."
```

Comment:
Is intent to disable the command until software is upgraded ? If so, I think this can safelly be removed.

Reply:
Just to make sure, do you mean that the `Write-Host` line can be removed since it is simply a redundant output?

Reply:
No, I wonder the purpose of entire script.

Reply:
The shimming needs to be removed after the uninstall. I agree that it is not necessary during update. However, do you think it is more appropriate to put these in the `chocolateyUninstall.ps1`? Or would the shimming executable automatically be deleted if no uninstallation script is present? Thanks!

Reply:
> Or would the shimming executable automatically be deleted if no uninstallation script is present? Thanks!

it would if you hadn't called `Install-BinFile`. Since everything is extracted to the package tools directory chocolatey will shim every `.exe` file automatically (this also removing them on uninstall). This isn't happening when you coll `Install-BinFile` manually though.

Reply:
Resolved in #1592.